app/gui2/src/stores/graph.ts

```diff
@@ -30,7 +34,7 @@ export const useGraphStore = defineStore('graph', () => {
   const metadata = computed(() => proj.module?.doc.metadata)
 
   const textContent = ref('')
-
+  const editedNodeInfo = ref<NodeEditInfo | null>()
```

Comment:
By default, `ref<NodeEditInfo>()` gives you a type `Ref<NodeEditInfo | undefined>`, since you didn't provide the initial value. For that reason let's use `undefined` instead of `null` here.